### PR TITLE
Moved path into common variable

### PR DESCRIPTION
**Pull request by radarhere**
Just a suggestion.

### CODE REVIEW

File: auto/run.sh [hunk 1]
```diff
@@ -37,68 +37,70 @@ pushd Pillow
     "$@"
   }
 
+  testsuiterun=../../testsuite/run.py
```
Comment (homm) [inline auto/run.sh]: Shoul be with underscore, i guess

Comment (radarhere) [reply auto/run.sh]: Ah, yes, thanks.

File: auto/run.sh [hunk 2]
```diff
@@ -37,68 +37,70 @@ pushd Pillow
     "$@"
   }
 
+  testsuite_run=../../testsuite/run.py
```
Comment (homm) [inline auto/run.sh]: Actually, I think something simpler and shorter is preferable. Maybe just `$RUN`?

Comment (radarhere) [reply auto/run.sh]: Okay, sure. Updated.

**Pull request merged**